.github/ISSUE_TEMPLATE/refactor-issue-template.md

```
@@ -0,0 +1,42 @@
+---
+name: Refactor issue template
+about: For refactor issue 
+title: 'refactor: '
+labels: "\U0001F9F9 Refactor"
+assignees:
+---
+
+### 1.  Issue Description
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)